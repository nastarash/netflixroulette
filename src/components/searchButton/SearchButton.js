import React from 'react';

class SearchButton extends React.Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  onButtonClick() {
    document.getElementById('resultForm').style.display = 'block';
  }

  render() {
    return (
      <button type="submit" onClick={this.onButtonClick}>
        Seacrh
      </button>
    );
  }
}

export default SearchButton;
