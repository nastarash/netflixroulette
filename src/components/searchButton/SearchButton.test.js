import React from 'react';
import renderer from 'react-test-renderer';
import SearchButton from './SearchButton';

it('render correctly', () => {
  const AppComponent = renderer.create(<SearchButton />).toJSON();
  expect(AppComponent).toMatchSnapshot();
});
