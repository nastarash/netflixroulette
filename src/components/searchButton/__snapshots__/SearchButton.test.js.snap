// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
Array [
  <div>
    <p>
      FIND YOUR MOVIE
    </p>
    <input
      type="text"
    />
  </div>,
  <form>
    <p>
      SEARCH BY
    </p>
    <div>
      <input
        className="toggle toggle-left"
        id="toggle-on"
        name="toggle"
        type="radio"
        value="false"
      />
      <label
        className="btn"
        htmlFor="toggle-on"
      >
        Title
      </label>
      <input
        className="toggle toggle-right"
        id="toggle-off"
        name="toggle"
        type="radio"
        value="true"
      />
      <label
        className="btn"
        htmlFor="toggle-off"
      >
        Genre
      </label>
    </div>
  </form>,
  <button
    onClick={[Function]}
    type="submit"
  >
    Seacrh
  </button>,
]
`;
